.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files

```

Comment:
Several `.idea/*` files seem to have been added?

Reply:
Oh this is the .idea directory created by the IDE, fixed it.